crates/sozo/src/commands/migrate.rs

```diff
@@ -36,16 +37,9 @@ pub struct MigrateArgs {
 }
 
 impl MigrateArgs {
-    pub fn run(mut self, config: &Config) -> Result<()> {
+    pub fn run(self, config: &Config) -> Result<()> {
         let ws = scarb::ops::read_workspace(config.manifest_path(), config)?;
 
-        // If `name` was not specified use package name from `Scarb.toml` file if it exists
-        if self.name.is_none() {
```

Comment:
hmm do we want to remove this?

Reply:
my bad, confused this with `WorldOptions` and thought we already do this internally will revert.